modules/commons/src/main/resources/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.2.json

```diff
@@ -0,0 +1,164 @@
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:BrainRegionOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a brain region",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:StimulusTypeOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a stimulus type",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:EcodeOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing an eCode",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:RoleOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a role",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:LicenseOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a license",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:SomaTypeTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing the types of somas of the reconstructed cell",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:UnitOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a unit code",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:LabelingCompoundOntologyTermShape",
+      "@type": "sh:NodeShape",
+      "label": "The shape of an ontology term representing a labeling compound",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/labeledontologyentity/v0.1.1/shapes/LabeledOntologyEntityShape"
+        }
+      ]
+    },
+    {
+      "@id": "this:MorphologicalCellTypeOntologyTermShape",
```

Comment:
A cell type ontology term shape is defined in this schema: "this:CellOntologyTermShape". There is a typo in the name. It should be ""this:CellTypeOntologyTermShape"". Do you mind update it too ?

Reply:
Updated

Reply:
Thank you.